composer.json

```diff
@@ -36,7 +36,7 @@
         "behatch/contexts": "3.1.0",
         "doctrine/annotations": "^1.2",
         "doctrine/doctrine-bundle": "^1.8",
-        "doctrine/orm": "^2.5.2",
+        "doctrine/orm": "^2.6.2",
```

Comment:
Isn't 2.6.0 enough? It would be better to set as minimal version the first Doctrine version supporting this feature.

Reply:
@dunglas yes. I edit composer.json